README.md

```diff
@@ -1,24 +1,49 @@
-# Kailua
+# OP KAILUA
```

Comment:
Is "OP Kailua" accurate? 
I think this repo also includes "Base Kailua" and "Worldchain Kailua"

Reply:
/shrug probably fine to label it as OP Kailua even if one might nit at the precision :)

Reply:
^ I do think just "Kailua" is better.